### PR TITLE
ALBS-40: Migrate PES from SQLite to PostgreSQL

### PR DESCRIPTION
**Pull request by soksanichenko**
- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn

### CODE REVIEW

File: src/backend/db/utils.py [hunk 1]
```diff
@@ -16,35 +16,36 @@
...
-@event.listens_for(SAEngine, "connect")
-def set_sqlite_pragma(dbapi_connection, connection_record):
-    cursor = dbapi_connection.cursor()
-    cursor.execute("PRAGMA foreign_keys=ON")
-    cursor.close()
+# @event.listens_for(SAEngine, "connect")
+# def set_sqlite_pragma(dbapi_connection, connection_record):
+#     cursor = dbapi_connection.cursor()
+#     cursor.execute("PRAGMA foreign_keys=ON")
+#     cursor.close()
```
Comment (anfimovdm) [inline src/backend/db/utils.py]: Is that mistake?

Comment (soksanichenko) [reply src/backend/db/utils.py]: Yep, I removed it.

**Review by anfimovdm — APPROVED** [review]

File: ci/Dockerfile.backend [hunk 2]
```diff
@@ -1,4 +1,5 @@
 FROM python:3 AS python39-uwsgi
+RUN apt-get install -y libpq-dev
```
Comment (rpisarev) [inline ci/Dockerfile.backend:3]: Maybe we should update the repo before installing and apt-get clean all after?

File: ci/Dockerfile.postgres [hunk 3]
```diff
@@ -0,0 +1,3 @@
+FROM postgres:12.3 as postgres-with-pgloader
+RUN apt-get update
+RUN apt-get install -y wget sudo pgloader libpq-dev
```
Comment (rpisarev) [inline ci/Dockerfile.postgres:3]: And removing a garbage again

**Pull request merged**